release_manifests/24.10-0.7.0.0-0.yaml

```diff
@@ -0,0 +1,45 @@
+precompiled:
+- 24.10-0.7.0.0-0-5.15.0-25-generic-ubuntu22.04-amd64
```

Comment:
why do we need to specify full version?

Reply:
the `24.10-0.7.0.0-0`?
it could be derived from the filename (and from the parameter value in the job parsing it), but having it here feels clearest and is explicit.

Reply:
I still think that we don'n need to duplicate version

Reply:
+1 on removing the prefix. Can be understood from filename.